resources/analysisTools/exomePipeline/targetExtractCoverageSlim.sh

```diff
@@ -85,9 +85,15 @@ ${RSCRIPT_BINARY} ${TOOL_ON_TARGET_COVERAGE_PLOTTER_BINARY} ${FILENAME_TARGETS_W
     && mv ${TARGETS_PLOT}.tmp ${TARGETS_PLOT} \
     || throw 15 "Error from on target coverage plotter"
 
+groupLongAndShortChromosomeNames "$FILENAME_GENOME_COVERAGE" \
```

Comment:
Put it in a separate file / method ? It's called several times right?

Reply:
See below. We don't safe much space and gain little clarity.